src/python/gi/overrides/BlockDev.py

```diff
@@ -262,7 +262,7 @@ def __init__(self, *args, **kwargs):   # pylint: disable=unused-argument
 __all__.append("CryptoLUKSExtra")
 
 class CryptoKeyslotContext(BlockDev.CryptoKeyslotContext):
-    def __new__(cls, passphrase=None, keyfile=None, keyfile_offset=0, key_size=0):
+    def __new__(cls, passphrase=None, keyfile=None, keyfile_offset=0, key_size=0, keyring=None):
         if (passphrase and keyfile):
```

Comment:
should now check `keyring` here too, right?

Reply:
Fixed.